en/mkdocs.yml

```diff
@@ -719,4 +720,4 @@ extra:
         - type: linkedin
           link: https://www.linkedin.com/company/wso2
     site_version: 3.2.0
-    base_path: https://apim.docs.wso2.com/en/3.2.0
+    base_path: http://localhost:8000
```

Comment:
This change is incorrect. The base_path should be "base_path: https://apim.docs.wso2.com/en/3.2.0"

Reply:
fixed